Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+
+WORKDIR /tmp/
+RUN curl -L https://github.com/NREL/Radiance/releases/download/5.2/radiance-5.2.dd0f8e38a7-Linux.tar.gz | tar xz
+RUN mv `ls`/usr/local/radiance/bin/* /usr/local/bin
+RUN mv `ls`/usr/local/radiance/lib/* /usr/local/lib
+RUN rm -rf `ls`
+
+WORKDIR /usr/app/
+# COPY requirements.txt requirements.txt
+# RUN pip install -r requirements.txt
+COPY . .
+RUN pip install -e .[cli]
+ENTRYPOINT [ "honeybee-radiance" ]
```

Comment:
This might limit the use of command line since you might want to use `honeybee` commands insider this docker image too. I suggest to change it to "honeybee" or does not include an entry point. With "honeybee" as entrypoint one can use `honeybee radiance` to access honeybee-radiance commands.

Reply:
I reckon we should keep it that way. That way there is a clear distinction between the `honeybee-radiance` and `honeybee-core` docker containers.

Reply:
I like the distinction idea but thinking about it more having an entrypoint will break the ability to run the same Queenbee workflow locally. 

This will make the command to be executed in docker to change from `honeybee-radiance grid split ...` to `grid split ...`. We should either adjust Queenbee to take two commands (it actually does still have an input for local) or remove the entrypoint so we can use an identical command for local runs and running them inside a docker container. 😵

Reply:
Aaaaaaaaaah I see what you mean now! Hmm... I'm torn about this one actually. Let's remove the entrypoint for now to make it easy and we can review that decision at a later date.